### PR TITLE
feat: adjust grid for mobile

### PR DESCRIPTION
**Pull request by sofiiadan**
Adjust space for mobile (not sure if this is all that is needed)

### CODE REVIEW

**Comment by Cyro292** [comment]
Looks Nice. 
Regarding Keys and Opportunities: 
There is a responsive layout issue observed when accessing the profile on devices with screens smaller than 500px in width. When the profile dropdown is activated, the size of the Table fluctuates, enlarging and shrinking dynamically. This behavior causes certain UI elements to be displaced, with some elements extending beyond the boundaries of the container.
 
image

image

**Comment by Cyro292** [comment]
If you have time left a few minor things:

In opportunities when having a screen <440px. 
1. The gap between the frame and card increases excessively as the screen size decreases. The spacing should remain consistent regardless of screen size (red)
2. Additional spacing is needed between the Button and the Header to improve layout clarity (blue).

image

When additing questionaires:
1. Increase the spacing between the 'Save' and 'Remove' buttons to prevent accidental clicks. Consider relocating the 'Remove' button or adding a confirmation dialog to enhance user experience and prevent unintended actions.

image

File: app/layout.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface Props {
 export default function RootLayout({ children }: Props) {
   return (
     <html lang="en">
-      <body>
+      <body className="">
```
Comment (BehsadRiemer) [inline app/layout.tsx]: this is a redundant change, can be taken out.

File: tailwind.config.ts
Comment (BehsadRiemer) [inline tailwind.config.ts:1]: These are the default breakpoint widths, not needed.

**Review by BehsadRiemer — CHANGES_REQUESTED** [review]
Overall, the changes are great and look good. Just some more fixes to make the UI more smooth. I will be commenting a summary.

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 22 41
The delete button seems to overflow, maybe consider putting this under "Review a candidate"

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 22 10

The questionnaire seems to be overflowing. Please consider making the size scale properly for mobile views.

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 45 12

The save and delete button can also be placed below the "Configure an existing opportunity"

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 22 25

"Start new review" is overflowing here and can be placed below "see and edit your reviews" for mobile views.

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 21 52

The webhook URL can be bigger.

**Comment by BehsadRiemer** [comment]
Screenshot 2024-06-21 at 11 20 15

Export data can be placed below " applications for {opportunity name}"

File: app/opportunities/[opportunity_id]/applications/[application_id]/page.tsx [hunk 2]
```diff
@@ -14,27 +15,25 @@ export default async function ApplicationOverview({
     },
   });
 
-
-  const opportunityTitle = db.opportunity.findUnique({
-    where: {
-      id: application?.opportunityId,
-    },
-  }).then((opportunity) => opportunity?.title);
+  const opportunityTitle = db.opportunity
```
Comment (BehsadRiemer) [inline app/opportunities/[opportunity_id]/applications/[application_id]/page.tsx]: I know you didn't write this but please make this a TRPC query

File: app/opportunities/[opportunity_id]/edit/editOpportunityForm.tsx
Comment (BehsadRiemer) [inline app/opportunities/[opportunity_id]/edit/editOpportunityForm.tsx:1]: Screenshot 2024-07-10 at 00 46 44

The description should be below for desktop views

File: components/header/index.tsx [hunk 3]
```diff
@@ -10,17 +10,22 @@ async function Header() {
...
-        )}
+      <div className="flex justify-between px-4 py-4">
+        <div className="flex items-center">
+          <Navigation />
+        </div>
+        <div className="flex items-start">
+          {session?.user ? (
+            <UserComponent user={session.user} />
+          ) : (
+            <Button asChild variant="outline" size="sm">
```
Comment (BehsadRiemer) [inline components/header/index.tsx]: sign in button styling is now different, please ensure it stays consistent with the dev branch

**Review by BehsadRiemer — CHANGES_REQUESTED** [review]
LGTM, some minor changes that I added under comments and conflict resolutions then you're done :)

**Review by BehsadRiemer — CHANGES_REQUESTED** [review]
Looks great, but the description field in the creation and editing of an opportunity is still not consistent with the original design (please see pictures). 

Google Chrome 2024-08-10 22 37 58

Please also make sure the padding of the titles is consistent:
Google Chrome 2024-08-10 22 41 24

**Pull request closed without merging**